cli_tools_tests/e2e/gce_ovf_import/test_suites/ovf_instance_import/ovf_instance_import_test_suite.go

```diff
@@ -89,6 +89,8 @@ func TestSuite(
 			testSuiteName, fmt.Sprintf("[%v] %v", testType, "Ubuntu 1604 from Virtualbox"))
 		instanceImportUbuntu16FromAWS := junitxml.NewTestCase(
 			testSuiteName, fmt.Sprintf("[%v] %v", testType, "Ubuntu 1604 from AWS"))
+		instanceImportWithNetworkTags := junitxml.NewTestCase(
```

Comment:
Add tags to an existing test to avoid expanding the number of tests we run.

Reply:
done! Added tags to the "network setting" tests, since the tags are network related.